.github/workflows/libvsync-actions.yml

```diff
@@ -0,0 +1,23 @@
+name: libvsync tests
+run-name: ${{ github.actor }} is testing out GitHub Actions 🚀
+on: [push]
+jobs:
+  Run-Tests:
+    runs-on: ubuntu-latest
+    steps:
+      - run: echo "🎉 The job was automatically triggered by a ${{ github.event_name }} event."
+      - run: echo "🐧 This job is now running on a ${{ runner.os }} server hosted by GitHub!"
+      - run: echo "🔎 The name of your branch is ${{ github.ref }} and your repository is ${{ github.repository }}."
+      - name: Check out repository code
+        uses: actions/checkout@v4
+      - run: echo "💡 The ${{ github.repository }} repository has been cloned to the runner."
+      - run: echo "🖥️ The workflow is now ready to test your code on the runner."
+      - name: Run all tests
+        run: mkdir build && cd build && cmake .. && make && ctest
```

Comment:
Can we split this step into 3 steps?:
1. configure: cmake -S. -Bbuild
2. cmake --build build
3. ctest --test-dir build

It would make it visible at a single glance in the github action UI which of these steps failed.

Reply:
I agree for the next version. I just want a super simple naive one at the moment.